techniques/system/common/1.0/cf-served.st

```diff
@@ -81,7 +81,7 @@ bundle server access_rules
   &endif&
 
 &endif&
-    policy_server_set::
+
```

Comment:
Using this class is no longer necessary, as the new implementation will *always* have a policy_server defined - which may be "rudder", but it will have one.